.codecov.yml

```diff
@@ -1,19 +1,29 @@
 comment: off
 ignore:
-  - "**/cbor_gen.go"
-  - "api/test/**/*"
-  - "api/test/*"
-  - "gen/**/*"
-  - "gen/*"
-  - "cmd/lotus-shed/*"
-  - "cmd/tvx/*"
-  - "cmd/lotus-pcr/*"
-  - "cmd/tvx/*"
-  - "cmd/lotus-chainwatch/*"
-  - "cmd/lotus-health/*"
-  - "cmd/lotus-fountain/*"
-  - "cmd/lotus-townhall/*"
-  - "cmd/lotus-stats/*"
-  - "cmd/lotus-pcr/*"
+  # Auto generated
+  - "**_gen.go$"
+  - "**/mocks_full.go$"
+  - "^chain/actors/builtin/[^/]*/(message|state|v)[0-4]\\.go$" # We test the latest version only.
```

Comment:
nit: it would be cool if this '4' was kept in a variable somewhere that got auto-updated by the actors release gen code.

Reply:
That would be nice, but maintaining this file with code-gen sounds like a pain.